NDB.Covid19/NDB.Covid19.Droid/Views/AuthenticationFlow/InformationAndConsentActivity.cs

```diff
@@ -33,6 +34,7 @@ public class InformationAndConsentActivity : AppCompatActivity
         TextView _consentExplanationText;
         Button _idPortenButton;
         InformationAndConsentViewModel _viewModel;
+        private TemporaryExposureKey.Types.ReportType _reportInfectedType;
```

Comment:
Looks a bit weird, maybe import whole `using NDB.Covid19.ProtoModels.TemporaryExposureKey.Types` with static, e.g., like on the line 17?